packages/framework/tinylicious-client/src/TinyliciousClient.ts

```diff
@@ -136,7 +139,7 @@ export class TinyliciousClient {
 		};
 	}
 
-	private createLoader(schema: ContainerSchema) {
+	private createLoader(schema: ContainerSchema): Loader {
```

Comment:
Seems like this could do away with returning `IHostLoader` instead. I know it's private, but still seems preferable?

Reply:
I'll take a look.

Reply:
Updated :)